performance_test/src/experiment_configuration/experiment_configuration.cpp

```diff
@@ -390,7 +395,9 @@ boost::uuids::uuid ExperimentConfiguration::id() const
 
 void ExperimentConfiguration::log(const std::string & msg) const
 {
-  std::cout << msg << std::endl;
+  if (false == m_is_drivepx_rt) {
+    std::cout << msg << std::endl;
```

Comment:
All the logging is done in a separate thread, is it still necessary to disable `cout`? Can you document why this is necessary?

Reply:
Since the test case do not have a way to assign separate low priority for this thread, all threads are  at the same priority level. Till then I would keep console output off for RT testings